docs/implementation/computation.rst

```diff
@@ -0,0 +1,187 @@
+===========
+Computation
+===========
+
+Fitting Bayesian statistical models of large metabolic networks poses
+distinctive computational problems. This document explains what the problems
+are and how Maud attempts to address them.
+
+Goal
+====
+
+Maud aims to fit Bayesian models of metabolic networks with about 100 internal
+reactions, each with its kinetics represented by the modular rate law and MWC
+formalism as described in the section on :doc:`Maud's statistical model
+<../theory/statistics>`, and around 20 experiments.
+
+Inputs
+======
+
+There should be informative prior information about all kinetic parameters,
+compound formation energies and unbalanced metabolite
+concentrations. Compartment volumes, measurements and measurement errors and a
```

Comment:
Should we make an issue for priors on kinetic parameters if we don't have any values?

Reply:
Good idea: #245